kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -95,7 +96,7 @@ public Subscriber execute(Query query) {
                 if (fqr.size() == 1) {
                     return (Subscriber) fqr.get(0);
                 }
-                throw new IllegalStateException(String.format(KilkariConstants.MORE_THAN_ONE_SUBSCRIBER, callingNumber));
+                throw new IllegalStateException(String.format(KilkariConstants.MORE_THAN_ONE_SUBSCRIBER_WITH_SAME_NUMBER, callingNumber));
```

Comment:
I want to make sure this exception being thrown is handled in our mcts import code as well. Might be unrelated to this feature PR though.